back/src/Guide/Config.hs

```diff
@@ -42,6 +42,8 @@ data Config = Config {
   _portMain      :: Int,          -- ^ Port for the main site.
   _portApi       :: Int,          -- ^ Port for the API.
   _portEkg       :: Int,          -- ^ Port for EKG stats.
+  _portMatomo    :: Maybe Int,    -- ^ Port for Matomo statistic.
+  _pageMatomo    :: Maybe Text,   -- ^ Page on Matomo to send statisic.
```

Comment:
Matomo will not necessarily be running on localhost.

We want just `_matomo :: Maybe Url`, there's no reason to separate the port and the page.